snippets/vs_code_snippets.md

```diff
@@ -0,0 +1,421 @@
+## VS Code Dart 1 Transformer Compatible Code Snippets 
```

Comment:
Are these strictly dart 1 compat code snippets? I'm wondering if it's even worth including.

Reply:
They are, I debated that myself and thought I would include since the readme currently links to Jace's vs code snippet repo. Maybe we leave them in there for now and officially remove them once the Dart 2 release of over_react is ready to publish?

Reply:
Having said that there is no point in really keeping them now since the transformer is already compatible with the builder boiler plate changes so maybe removing them entirely now is the right move.

Reply:
I'm going to remove them.